extensions/authclient/BaseOAuth.php

```diff
@@ -397,7 +397,7 @@ protected function composeUrl($url, array $params = [])
     /**
      * Saves token as persistent state.
      * @param OAuthToken $token auth token
-     * @return static self reference.
+     * @return $this
```

Comment:
I was thinking. If I put

    @return Object

The api doc will change it to a link pointing to the documentation for object.

Can we do the same for `$this`?